src/api/error.rs

```diff
@@ -89,15 +89,15 @@ pub enum ApiError {
     #[error("JSON encoding failed")]
     JsonEncodingFailed(String, serde_json::Error),
 
+    // SDK error.
+    #[error("an unexpected SDK error has occured.")]
```

Comment:
```suggestion
    #[error("an unexpected SDK error has occured")]
```

The messages intentionally start with lower case and don't end with `.` to allow the printer to control formatting.